components/frontend-react/src/components/Appbar.js

```diff
@@ -62,6 +64,7 @@ const BassaAppBar = (props) => {
     switch(route){
       case 'logout':
         sessionStorage.removeItem('token');
+        props.logoutUser()
         props.history.push('/');
```

Comment:
You need to clean the history and the push the basepath. Try replace. Read the history documentation.